src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12718,38 +12718,41 @@ void Unit::SetInCombatWith(Unit* enemy, uint32 duration)
     SetInCombatState(false, enemy, duration);
 }
 
-void Unit::CombatStart(Unit* target, bool initialAggro)
+void Unit::CombatStart(Unit* vicitm, bool initialAggro)
```

Comment:
Shouldn't be `victim` instead of `vicitm`?

Reply:
Ah yes typo, thanks